CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [v0.4.0]
+
+### Added
+
+- Added options to monitor configmap and secret resources to determine staticClients.
+
```

Comment:
- Support Multiple redirection URL on a single static client by using a `comma separated list` as annotation value

Reply:
Also add link for diff of commits 

[v0.3.1..v0.4.0](https://github.com/mintel/dex-k8s-ingress-watcher/compare/v0.3.1...v0.4.0)